apps/site/src/app/(home)/sections/Landing/Landing.tsx

```diff
@@ -0,0 +1,19 @@
+import Button from "@/components/Button/Button";
+import styles from "./Landing.module.css";
+
+const Landing = () => {
+	return (
+		<section className={styles.landingSection}>
+			<div className={styles.landingGroup}>
+				<h1 className="font-display">IrvineHacks 2024</h1>
+				<p className="font-display">January 26-28</p>
```

Comment:
Nitpick: although we don't have an official style guide, we tend to prefer using n-dashes (e.g. 26–28, `&ndash;`) for ranges.